test/helpers-asHTML.test.ts

```diff
@@ -138,6 +138,17 @@ it("includes target attribute on links with a target value", () => {
 	)
 })
 
+it("includes `dir` attribute on right-to-left languages", () => {
+	const richTextFixtures = createRichTextFixtures()
+
+	expect(asHTML(richTextFixtures.ar, { linkResolver })).toMatchSnapshot()
+
+	// TODO: Remove when we remove support for deprecated tuple-style configuration.
+	expect(asHTML(richTextFixtures.ar, linkResolver)).toBe(
+		asHTML(richTextFixtures.ar, { linkResolver }),
+	)
```

Comment:
💡 #idea: I think we can remove this extra `expect`. The test checks for `dir`, which shouldn't be affected by the link resolver.

Reply:
Will do!